client/ClientNetworking.cpp

```diff
@@ -118,6 +118,7 @@ namespace {
                                 this,
                                 boost::asio::placeholders::error,
                                 boost::asio::placeholders::bytes_transferred));
+
```

Comment:
Is this whitespace intentional or just slipped in?

Reply:
it was added intentionally between when I replaced all the `boost::bind` and when I reverted most of the networking ones that use `asio::placeholders` such as the above.